include/cantera/oneD/IonFlow.h

```diff
@@ -28,9 +28,14 @@ namespace Cantera
 class IonFlow : public Flow1D
 {
 public:
+    //! Create a new IonFlow domain.
+    //! @param ph Object representing the gas phase. This object will be used
```

Comment:
Nitpick: two spaces after parameter names makes things more readable … not a major issue overall.